sdk/metricsadvisor/Azure.AI.MetricsAdvisor/src/Models/DataFeed/DataFeed.cs

```diff
@@ -14,34 +14,11 @@ namespace Azure.AI.MetricsAdvisor.Models
     /// </summary>
     public class DataFeed
     {
-        private IList<string> _administrators;
-
-        private IList<string> _viewers;
-
         /// <summary>
         /// Initializes a new instance of the <see cref="DataFeed"/> class.
         /// </summary>
-        /// <param name="dataFeedName">A custom name for the <see cref="DataFeed"/>.</param>
-        /// <param name="dataSource">The source from which data is consumed.</param>
-        /// <param name="dataFeedGranularity">The frequency with which ingestion from the data source occurs.</param>
-        /// <param name="dataFeedSchema">Defines how this <see cref="DataFeed"/> structures the data ingested from the data source in terms of metrics and dimensions.</param>
-        /// <param name="dataFeedIngestionSettings">Configures how a <see cref="DataFeed"/> behaves during data ingestion from its data source.</param>
-        /// <exception cref="ArgumentNullException"><paramref name="dataFeedName"/>, <paramref name="dataSource"/>, <paramref name="dataFeedGranularity"/>, <paramref name="dataFeedSchema"/>, or <paramref name="dataFeedIngestionSettings"/> is null.</exception>
-        /// <exception cref="ArgumentException"><paramref name="dataFeedName"/> is empty.</exception>
-        public DataFeed(string dataFeedName, DataFeedSource dataSource, DataFeedGranularity dataFeedGranularity, DataFeedSchema dataFeedSchema, DataFeedIngestionSettings dataFeedIngestionSettings)
+        public DataFeed()
         {
-            Argument.AssertNotNullOrEmpty(dataFeedName, nameof(dataFeedName));
-            Argument.AssertNotNull(dataSource, nameof(dataSource));
-            Argument.AssertNotNull(dataFeedGranularity, nameof(dataFeedGranularity));
-            Argument.AssertNotNull(dataFeedSchema, nameof(dataFeedSchema));
-            Argument.AssertNotNull(dataFeedIngestionSettings, nameof(dataFeedIngestionSettings));
-
-            Name = dataFeedName;
-            DataSource = dataSource;
-            SourceType = dataSource.Type;
-            Granularity = dataFeedGranularity;
-            Schema = dataFeedSchema;
-            IngestionSettings = dataFeedIngestionSettings;
             Administrators = new ChangeTrackingList<string>();
```

Comment:
could u remind me the purpose of the `ChangeTrackingList` type? I am not familiar with it

Reply:
It's useful to differentiate between `make this list empty` and `don't change this list`. Suppose our data feed has some viewers, and we want to update its description without changing the list of viewers.

If we do:
```cs
var dataFeed = new DataFeed() { Description = "What a great description." };
client.UpdateDataFeed(dataFeed);
```
Even though `dataFeed.Viewers` is empty, this Update won't change the Viewers list because we didn't touch it. The client won't send it.

If we did:
```cs
var dataFeed = new DataFeed();
dataFeed.Viewers.Clear();

client.UpdateDataFeed(dataFeed);
```
This would clear the list on the service side as well, since we touched it.